assets/src/edit-story/output/page.js

```diff
@@ -70,8 +76,17 @@ function OutputPage({ page }) {
       element.id !== backgroundElementId &&
       element.link?.type === LinkType.ONE_TAP
   );
+  const longestMediaElement = getLongestMediaElement(elements);
+
   return (
-    <amp-story-page id={id}>
+    <amp-story-page
+      id={id}
+      auto-advance-after={
+        autoAdvance
+          ? longestMediaElement?.id || `${defaultPageDuration}s`
```

Comment:
If the longest video is super short, e.g. 1s, shouldn't it use the `defaultPageDuration`? Generally, if the longest media is shorter than the set duration, should it use that instead?

Reply:
@pbakaus WDYT?

Reply:
mhh...good point. I do think that if somebody drags in a ~4 seconds video, and the page duration is set to 7s, they will want auto advance after the video (a frozen video almost always looks strange). However, I can see the case for small, decorative 'transition in' videos.

For now, let's ship with this implementation, but paging @samitron7 to chime in, and possibly design a per-video override, e.g. a checkbox in the video settings that says something like "Decorative (does not trigger auto advancing)". We can also try a certain threshold (if video is less than 1/3 of page duration, it will not auto advance), but these all seem unsafe.